src/styles/_fonts.scss

```diff
@@ -1,6 +1,128 @@
 @font-face {
```

Comment:
you need only a few fonts in this project, but not all of this

Reply:
I think, for future, if we want to update project, we will use these fonts and we willn't find them secondly.